### PR TITLE
Public concordances api

### PR DESCRIPTION
**Pull request by dgem**

### CODE REVIEW

**Comment by Zapadlo** [comment]
+1

File: public-concordances-api-sidekick@.service [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[Service]
+ExecStart=/bin/sh -c '\
+  etcdctl set   /ft/services/public-concordances-api/healthcheck     true; \
+  etcdctl mkdir /ft/services/public-concordances-api/servers; \
+  etcdctl set 	/ft/healthcheck/public-concordances-api-%i/path /__health; \
+  etcdctl set 	/ft/healthcheck/public-concordances-api-%i/categories read; \
+  etcdctl set 	/vulcand/frontends/public-services-concordances/frontend \'{"Type": "http", "BackendId": "vcb-public-concordances-api", "Route": "PathRegexp(`/concordances/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$`) && Host(`public-services`)"}\'; \
+  while true; do \
+    PORT=`[ $PORT ] && echo $PORT || echo $(/usr/bin/docker port public-concordances-api-%i 8080 | cut -d":" -f2)`; \
+    curl -s -H "Accept: application/json" 0.0.0.0:$PORT/__health | jq -e \'[.checks[].ok] | all\' 2>&1>/dev/null && etcdctl set /ft/services/public-concordances-api/servers/%i http://%H:$PORT --ttl 5; \
```
Comment (catalin-ilea) [inline public-concordances-api-sidekick@.service]: Can you increase the ttl to 10? seems to help
https://github.com/Financial-Times/up-service-files/pull/470/files

Comment (dgem) [inline public-concordances-api-sidekick@.service]: Have increased ttl to 10 and left sleep at 5

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**